front-end/package-lock.json

```diff
@@ -1207,262 +1207,1338 @@
         }
```

Comment:
Kodėl pas tave šis failas taip skiriasi nuo mūsų? Pabandyk padaryt npm i ir pažiūrėk, kaip pasikeis dependencies